components/prism-orfeo.js

```diff
@@ -0,0 +1,39 @@
+(function (Prism) {
+	Prism.languages.orfeo = {
+		'parenthesis_open': {
+			pattern: /(^|\(|\^|\s)\((?=[^\.]|$)/,
+			alias: 'punctuation',
+			lookbehind: true
+		},
+		'parenthesis_close': {
+			pattern: /(^|[^^])\)(?=\)|\.|\s|$)/,
+			alias: 'punctuation',
+			lookbehind: true
+		},
+		'duration': {
+			pattern: /(^|\(|\s):(?:=|(?:\.[0-9]+|(?:0|[1-9][0-9]*)(?:\.[0-9]*)?)(?:\/(?:0?\.[0-9]*[1-9][0-9]*|[1-9][0-9]*(?:\.[0-9]*)?))?)(?=\)|\s|$)/,
```

Comment:
resolved in 83ec30b 

The reason for that pattern is to avoid coloring divisions by zero. For that reason, I'd rather use `0?\.0*[1-9]\d*` than removing the "at least one non-zero digit" requirement.